modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/endpoint/access/token/grant/handler/LiferayJWTBearerGrantHandler.java

```diff
@@ -162,8 +174,16 @@ protected boolean isGrantHandlerEnabled() {
 		return _oAuth2ProviderConfiguration.allowJWTBearerGrant();
 	}
 
+	private <U, V> void _addDefaults(Map<U, V> map, Map<U, V> defaultsMap) {
+		if (defaultsMap != null) {
+			defaultsMap.forEach(map::putIfAbsent);
+		}
+
+		_rebuild(CompanyConstants.SYSTEM);
```

Comment:
I cannot bring myself to believe that we need this line, but I am also not confident enough to make a commit to remove it. So I am leaving a comment here. @stian-sigvartsen